server/api.go

```diff
@@ -114,7 +114,6 @@ func (a *API) processActivity(w http.ResponseWriter, req *http.Request) {
 			continue
 		}
 
-		a.p.GetMetrics().ObserveChangeEventTotal(activity.ChangeType)
```

Comment:
This is now handled by the activity handler (two metrics counting the same thing previously).